packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -678,17 +678,15 @@ export class EngineReportingAgent<TContext = any> {
     executableSchemaId: string;
     executableSchema: string;
   }) {
-    this.logger.info('Starting schema reporter...')
-
-    this.logger.info(
-      `Schema reporter options: ${JSON.stringify(
-        {
-          overrideReportedSchema: this.options.experimental_overrideReportedSchema,
-          schemaReportingInitialDelayMaxMs: this.options.experimental_schemaReportingInitialDelayMaxMs,
-          schemaReportingUrl: this.options.schemaReportingUrl,
-        }
-      )}`
-    )
+    this.logger.info('Starting schema reporter...');
+    this.logger.info(`Schema reporting override is ${
+      this.options.experimental_overrideReportedSchema ?
+        'enabled' : 'disabled'
+    }`);
+    this.logger.info(`Schema reporting max initial delay: ${
```

Comment:
This could be undefined so I would first check if this is set before logging this. It would be confusing to see an empty or undefined here.

Reply:
Sounds good, I changed the phrasing as well to `Schema reporting max initial delay override` to indicate it's overriding a default.